src/infrastructure/AccountHttp.ts

```diff
@@ -118,11 +184,34 @@ export class AccountHttp extends Http implements AccountRepository {
      * @returns Observable<AccountInfo[]>
      */
     public getAccountsInfo(addresses: Address[]): Observable<AccountInfo[]> {
```

Comment:
:warning: not correct.

you should be copying over the nem2-library/src/api/* classes into the SDK because they are swagger generated.